### PR TITLE
Add release maven configurations.

### PR DESCRIPTION
**Pull request by wtritch**
This PR adds maven-release-plugin dependencies to the pom.xml files.

### CODE REVIEW

File: common/pom.xml [hunk 1]
```diff
@@ -7,7 +7,6 @@
     <name>tds-common</name>
 
     <packaging>jar</packaging>
-    <version>0.0.1-SNAPSHOT</version>
```
Comment (wtritch) [inline common/pom.xml:10]: If excluded, this module will inherit it's parent POM's version.

File: common/pom.xml [hunk 2]
```diff
@@ -138,12 +137,33 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
-                <version>3.5.1</version>
```
Comment (wtritch) [inline common/pom.xml:141]: Moved this version definition into a parent property.

File: common/pom.xml [hunk 3]
```diff
@@ -138,12 +137,33 @@
...
+                                <goals>
+                                    <goal>jar</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+    </profiles>
```
Comment (wtritch) [inline common/pom.xml:168]: I like being able to "download sources" when working with an external library.  Any objections to publishing source jars to the artifact repository?

File: pom.xml [hunk 4]
```diff
@@ -1,27 +1,84 @@
...
+    </scm>
+
     <modules>
         <module>common</module>
         <module>legacy</module>
     </modules>
 
     <properties>
-        <tds.assessment.client>0.0.1-SNAPSHOT</tds.assessment.client>
-        <tds.common>0.0.1-SNAPSHOT</tds.common>
```
Comment (wtritch) [inline pom.xml:19]: Removed some unused properties.

File: pom.xml [hunk 5]
```diff
@@ -1,27 +1,84 @@
...
     <name>tds-common-parent</name>
     <packaging>pom</packaging>
     <version>0.0.1-SNAPSHOT</version>
 
+    <scm>
+        <connection>scm:git:https://github.com/SmarterApp/TDS_Common.git</connection>
+        <developerConnection>scm:git:git@github.com:SmarterApp/TDS_Common.git</developerConnection>
+        <url>https://github.com/SmarterApp/TDS_Common</url>
+        <tag>HEAD</tag>
+    </scm>
```
Comment (wtritch) [inline pom.xml:15]: This is needed so the 'release:prepare' goal can push version bumps back into the release branch.

File: pom.xml [hunk 6]
```diff
@@ -1,27 +1,84 @@
...
+        </plugins>
+    </build>
+
+    <distributionManagement>
+        <repository>
+            <id>bintray-smarter-app-open-test-system</id>
+            <name>smarter-app-open-test-system</name>
+            <url>https://api.bintray.com/maven/smarter-app/open-test-system/${publish.package.name}/</url>
+        </repository>
+    </distributionManagement>
```
Comment (wtritch) [inline pom.xml:83]: This tells the 'release:perform' goal where to publish the versioned artifacts.

File: common/pom.xml (hunk 3 above)
Comment (millerts) [reply common/pom.xml:168]: I agree. Please publish the source jars.

**Review by millerts — APPROVED** [review]

**Review by jtreuting — APPROVED** [review]

File: common/pom.xml (hunk 3 above)
Comment (wtritch) [reply common/pom.xml:168]: Also, notice that I'm not including a source jar for the legacy project... think I should?

**Pull request merged**